packages/react-flex/src/components/Flex/useFlexStyles.ts

```diff
@@ -14,13 +57,43 @@ const useStyles = makeStyles({
  */
 export const useFlexStyles = (state: FlexState): FlexState => {
   const styles = useStyles();
-  state.className = mergeClasses(styles.root, state.className);
 
-  // TODO: Hook up slot styles, for example:
-  // const exampleSlotClassName = useExampleSlotStyles(state);
-  // if (state.exampleSlot) {
-  //   state.exampleSlot.className = mergeClasses(exampleSlotClassName, state.exampleSlot.className);
-  // }
+  const isRowDirection = state.direction?.substring(0, 3) === 'row';
+  state.className = mergeClasses(
+    styles.root,
+    state.direction === 'row' && styles.flexDirectionRow,
+    state.direction === 'column' && styles.flexDirectionColumn,
+    state.direction === 'row-reverse' && styles.flexDirectionRowReverse,
+    state.direction === 'column-reverse' && styles.flexDirectionColumnReverse,
```

Comment:
I am not entirely sure that we should support `-reverse` values as they are breaking narration: screen reader won't be able to read as per the provided direction, it will follow the DOM order.

Reply:
@smhigley can you please confirm if it is something that we should consider? 🤔

Reply:
Why is this a concern? Flex is supposed to act on visual layout.

Reply:
I think the problem might be that often what seems like a visual layout issue might actually be a rendering issue. This would affect screen readers, because they don't care about css or visual layout generally... DOM is their truth.

It's always easier to add smth than to remove it so we could consider dragging our users into a 'pit of success' 😅😅

I don't think it's a blocker. but would appreciate @smhigley's input